### PR TITLE
Adds notes for 4.10.5 release

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for Monday 22.03.21 z-stream release.

- Applies to `enterprise-4.10` only
- [Preview link](https://deploy-preview-43483--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-5)

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2590,3 +2590,29 @@ link:https://access.redhat.com/solutions/6815061[{product-title} 4.10.4 containe
...
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6828301[{product-title} 4.10.5 container image list]
+
+[id="ocp-4-10-5-known-issues"]
+==== Known issues
+
+* There is an issue adding a cluster through zero touch provisioning (ZTP) with the name `ztp*`. Add it causes a situation where `ArgoCD` deletes policies that ACM copies in to the cluster namespace. Consequently leading to a reconciliation loop, and the policies will not be compliant. As a workaround, do not name clusters with `ztp` at the beginning of the name. By adjusting the name of the cluster, collision will stop the reconciliation loop and the policies will be compliant. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049154[*BZ#2049154*])
```
Comment (tmalove) [inline release_notes/ocp-4-10-release-notes.adoc:2608]: Hi @opayne1 Replace the start of the second sentence with '**Adding** it causes a situation...'.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -2590,3 +2590,29 @@ link:https://access.redhat.com/solutions/6815061[{product-title} 4.10.4 containe
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-5"]
+=== RHBA-2022:0982 - {product-title} 4.10.4 bug fix and security update
```
Comment (tmalove) [inline release_notes/ocp-4-10-release-notes.adoc:2595]: Checking to see if there is a **{product-version}** for 4.**10**?

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc]: updated

**Review by jeana-redhat — COMMENTED** [review]
Some suggestions about the KI

**Pull request merged**